README.md

```diff
@@ -66,36 +66,6 @@ For your IDE to resolve all the files, run `yarn codegen && yarn build`.
    - Open a web browser and navigate to http://localhost:3000.
    - Use the provided interface to write and execute your GraphQL queries.
 
-### With multichain-testing
```

Comment:
why remove this?

Reply:
I needed it also for seeing other types of transactions.

note also that the opco-subql is a fork of this one so the more we can keep them the same the better. 

it doesn't cause any harm here and it's entirely accurate, so please retain it